libcxx/include/__memory/unique_ptr.h

```diff
@@ -129,6 +130,17 @@ class _LIBCPP_UNIQUE_PTR_TRIVIAL_ABI _LIBCPP_TEMPLATE_VIS unique_ptr {
 
   static_assert(!is_rvalue_reference<deleter_type>::value, "the specified deleter type cannot be an rvalue reference");
 
+  // A unique_ptr contains the following members which may be trivially relocatable:
+  // - pointer : this may be trivially relocatable, so it's checked
+  // - delter_type: this may be trivially relocatable, so it's checked
+  //
+  // This uniuqe_ptr implementation only contains a pointer to the unique object and a delter, so there are no
```

Comment:
`uniuqe_ptr` should be `unique_ptr`.(Twice)
`delter` should be `deleter`.(Twice)